lib/mysql.js

```diff
@@ -307,17 +310,43 @@ MySQL.prototype.updateOrCreate = function(model, data, options, cb) {
   this._modifyOrCreate(model, data, options, fields, cb);
 };
 
-function dateToMysql(val) {
-  return val.getUTCFullYear() + '-' +
-    fillZeros(val.getUTCMonth() + 1) + '-' +
-    fillZeros(val.getUTCDate()) + ' ' +
-    fillZeros(val.getUTCHours()) + ':' +
-    fillZeros(val.getUTCMinutes()) + ':' +
-    fillZeros(val.getUTCSeconds());
+function dateToMysql(dt, tz) {
+  if (!tz || tz == 'local') {
+    return dt.getFullYear() + '-' +
+      fillZeros(dt.getMonth() + 1) + '-' +
+      fillZeros(dt.getDate()) + ' ' +
+      fillZeros(dt.getHours()) + ':' +
+      fillZeros(dt.getMinutes()) + ':' +
+      fillZeros(dt.getSeconds());
+  } else {
+    tz = convertTimezone(tz);
+
+    if (tz !== false && tz !== 0) {
```

Comment:
Change this to: `if (tz !== false && tz !== 0) dt.setTime(dt.getTime() + (tz * 60000));`